oaknational/Oak-Web-Application
# Feat/pupil 1018/create new events from plan

## Description

Music year: 2017

Fire Avo events for following:
- LessonAbandoned
- LessonAccessed
- BrowseAccessed
- BrowseRefined

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2905--oak-web-application.netlify.thenational.academy
2. Click on pupil page 
3. You should see avo event fired for browse accessed;
4. click through browse experience
5. you should see browse refined events bengfired
6. click on lesson 
7. you should see lesson accessed event fired
8. clicked "back button"
9. you should see lesson abandoned event fired

note: ive not included browse abandomed as theres no back button to leave the browseX; optional to ass a ticket to add one in??

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
